kitsune/groups/views.py

```diff
@@ -92,6 +92,11 @@ def edit_avatar(request, group_slug):
         old_avatar_path = prof.avatar.name
 
     if request.method == "POST" and form.is_valid():
+        # If no avatar was uploaded, let's error out.
+        if "avatar" not in request.FILES:
```

Comment:
This should be moved to the clean method of the form. This way the `form.is_valid()` check will be erroneous and we won' have to add the next couple of lines.